packages/lesswrong/lib/collections/dialogueChecks/collection.ts

```diff
@@ -0,0 +1,36 @@
+import {ensureIndex} from "../../collectionIndexUtils";
+import {addUniversalFields, getDefaultMutations, getDefaultResolvers, schemaDefaultValue} from "../../collectionUtils";
+import {createCollection} from "../../vulcan-lib";
+import schema from "./schema";
+
+export const DialogueChecks: DialogueChecksCollection = createCollection({
+  collectionName: 'DialogueChecks',
+  typeName: 'DialogueCheck',
+  collectionType: 'pg',
+  schema,
+  resolvers: getDefaultResolvers('DialogueChecks'),
+  logChanges: true,
+})
+
+DialogueChecks.checkAccess = async (user: DbUser|null, document: DbDialogueCheck, context: ResolverContext|null): Promise<boolean> => {
+  // Case 1: A user can see their own checks
```

Comment:
You can do an early return where `if (!currentUser) return false;`, which eliminates any need to do null-safe handling later

Reply:
didn't import easily, ignoring

Reply:
@lagerros clarify?  this particular comment isn't critical but I'm going to push back against ignoring feedback, _especially_ if what you meant was that the feedback wasn't understood